kubernetes/host-logs-geneva.yaml

```diff
@@ -58,8 +61,8 @@ spec:
             command:
               - cmd
               - /c
-              - .\opt\hostlogswindows\scripts\cmd\livenessprobe.exe
-              - ".\\etc\\hostlogswindows\\filesystemwatcher.txt"
+              - "%CONTAINER_SANDBOX_MOUNT_POINT%\\opt\\hostlogswindows\\scripts\\cmd\\livenessprobe.exe"
```

Comment:
Yes, I believe this should be used in AKS RP as well. %CONTAINER_SANDBOX_MOUNT_POINT% is set in all HPCs. This naming structure makes the yaml compatible with both containerd 1.6 and 1.7. Otherwise, we'd need 2 separate yaml files depending on the containerd version